statistical_validation_run/artifacts.py

```
@@ -479,11 +486,13 @@ def render_report_md(
         "trade-tape volume at the order's own price.",
         "- Completion fills are priced at the ask — the optimistic end of taker "
         "(upper bound, noted in the design doc). The pessimistic sensitivity "
-        f"column re-prices that completion one tick worse plus `gas={PESSIMISTIC_CONVERSION_GAS:.2f}`; "
+        "column re-prices that completion one tick worse; "
         "GO requires the base AND the pessimistic `ci90_lower_pct` to pass.",
         "- Rebate income is `None` on graduated spread markets (`rebate_est`); "
         "the report never invents rebate income.",
-        "- Gas is 0 in shadow; `merge_gas_usd` is reported as the economic gate.",
+        "- Gas is 0, in shadow and live alike: merges are submitted through "
+        "Polymarket's relayer, which pays for them. The economic gate is "
+        "positive expectancy per close.",
```

Ruff (0.16.3)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)